src/renderer/pages/Dashboard.tsx

```diff
@@ -0,0 +1,116 @@
+    status: status ? Boolean(status) : undefined,
+  });
+
+  const {
+    data: projectsHostedCountData,
+    isLoading: projectsHostedCountLoading,
+    error: projectsHostedCountError,
+  } = useGetProjectsCountQuery({
+    hostRegistry: hostRegistry ? Boolean(hostRegistry) : undefined,
+  });
+
+  const {
+    data: issuedCarbonByMethodologyData,
+    isLoading: issuedCarbonByMethodologyLoading,
+    error: issuedCarbonByMethodologyError,
+  } = useGetIssuedCarbonByMethodologyQuery({});
+
+  const {
+    data: issuedCarbonByProjectTypeData,
+    isLoading: issuedCarbonByProjectTypeLoading,
+    error: issuedCarbonByProjectTypeError,
+  } = useGetIssuedCarbonByProjectTypeQuery({});
+
+  const contentsLoading = useMemo<boolean>(() => {
+    return (
+      projectsStatusCountLoading ||
+      projectsHostedCountLoading ||
+      issuedCarbonByMethodologyLoading ||
+      issuedCarbonByProjectTypeLoading
+    );
+  }, [
+    projectsStatusCountLoading,
+    projectsHostedCountLoading,
+    issuedCarbonByMethodologyLoading,
+    issuedCarbonByProjectTypeLoading,
+  ]);
+
+  if (contentsLoading) {
+    return <SkeletonProjectsView />;
+  }
+
+  if (
+    projectsStatusCountError ||
+    projectsHostedCountError ||
+    issuedCarbonByMethodologyError ||
+    issuedCarbonByProjectTypeError
+  ) {
+    return <FormattedMessage id={'unable-to-load-contents'} />;
+  }
+
+  if (
+    !projectsStatusCountData &&
+    !projectsHostedCountData &&
+    !issuedCarbonByMethodologyData &&
+    !issuedCarbonByProjectTypeData
+  ) {
+    return <FormattedMessage id={'no-records-found'} />;
+  }
+
+  return (
+    <>
+      {contentsLoading && <IndeterminateProgressOverlay />}
+      <Tabs aria-label="Default tabs" className="pt-4">
+        <Tabs.Item title={<FormattedMessage id="projects-view" />} id="projects-view">
+          {contentsLoading ? (
+            <SkeletonProjectsView />
+          ) : (
+            <ProjectsView
+              projectsStatusCountData={(projectsStatusCountData?.data as ProjectsStatusCount[]) || []}
+              projectsStatusCountLoading={projectsStatusCountLoading}
+              projectsHostedCountData={projectsHostedCountData?.data as ProjectsHostedCount}
+              projectsHostedCountLoading={projectsHostedCountLoading}
+              issuedCarbonByMethodologyData={
+                issuedCarbonByMethodologyData?.data || { issuedTonsCo2: [], context: 'Methodology' }
+              }
+              issuedCarbonByMethodologyLoading={issuedCarbonByMethodologyLoading}
+              issuedCarbonByProjectTypeData={
+                issuedCarbonByProjectTypeData?.data || { issuedTonsCo2: [], context: 'ProjectType' }
+              }
+              issuedCarbonByProjectTypeLoading={issuedCarbonByProjectTypeLoading}
+            />
+          )}
+        </Tabs.Item>
+
+        <Tabs.Item title={<FormattedMessage id="units-view" />} id="units-view">
+          <div className="p-4">
+            <UnitsView />
```

Comment:
this should be renamed units tab and moved to the components/blocks/tabs directory